bfv/decryptor.go

```diff
@@ -1,69 +1,25 @@
 package bfv
 
 import (
-	"github.com/ldsec/lattigo/v2/ring"
 	"github.com/ldsec/lattigo/v2/rlwe"
 )
 
-// Decryptor is an interface for decryptors
-type Decryptor interface {
```

Comment:
It is probably better to keep an interface at the scheme's level too.

Reply:
fixed